java/aiservices/openai/src/main/java/com/microsoft/semantickernel/aiservices/openai/textcompletion/OpenAITextGenerationService.java

```diff
@@ -116,24 +131,13 @@ private CompletionsOptions getCompletionsOptions(
     /**
      * Builder for a TextGenerationService
      */
-    public static class Builder {
-        private String modelId;
-        private OpenAIAsyncClient client;
-
-        public Builder withModelId(String modelId) {
-            this.modelId = modelId;
-            return this;
-        }
-
-        public Builder withOpenAIAsyncClient(OpenAIAsyncClient client) {
-            this.client = client;
-            return this;
-        }
+    public static class Builder extends TextGenerationService.Builder {
 
-        public OpenAITextGenerationService build() {
+        public TextGenerationService build() {
```

Comment:
This one returns the interface, but the OpenAIChatCompletion builder returns the class, should both return the same kind of object? either class or interface.

Reply:
both returning TextGenerationService I think is good